tests/neg/trailingCommas/trailingCommas.scala

```diff
@@ -0,0 +1,51 @@
+package foo
+
+// Multi-line only cases: make sure trailing commas are only supported when multi-line
+
+trait ArgumentExprs1 { f(23, "bar", )(Ev0, Ev1) } // error // error
+trait ArgumentExprs2 { f(23, "bar")(Ev0, Ev1, ) } // error // error
+trait ArgumentExprs3 { new C(23, "bar", )(Ev0, Ev1) } // error // error
+trait ArgumentExprs4 { new C(23, "bar")(Ev0, Ev1, ) } // error // error
```

Comment:
It would be nice to avoided undefined identifiers so that we only get errors about trailing commas. Same for the code below.